File: Fundamentals of C programming/step06/6.2/6.2.01.c
/*

Сравнение чисел
Напишите программу сравнивающие два целых числа.

Входные данные:
Два целых числа xx, yy

Выходные данные:
1 -- если x=yx=y
0 -- если x≠yx=y

Sample Input:

-2 2

Sample Output:

0

*/

#include <stdio.h>

int main() {
  int x,y;
  scanf("%d %d",&x,&y);
  
  switch (x==y) {
	case 0: printf("%d\n",0); break;
	case 1: printf("%d\n",1); break;
  }
  
  return 0;
}

File: Diving into JavaScript: For Beginners/step06/6.02/6.2.01.js
/*

Перед вами объект catalogue, в котором содержится свойство magazine: 5. 
Удалите это свойство из объекта и добавьте в объект свойство newspaper с числовым значением, которое пользователь вводит самостоятельно. 
Выведите значение свойства newspaper. 

Sample Input 1:

777

Sample Output 1:

777

Sample Input 2:

123

Sample Output 2:

123

Sample Input 3:

6

Sample Output 3:

6

*/

const catalogue = {
    magazine: 5,
};
delete catalogue.magazine;
let n = Number(prompt());
catalogue.newspaper = n;

console.log(catalogue.newspaper);
